mage_ai/server/constants.py

```diff
@@ -2,3 +2,5 @@
 
 SERVER_HOST = os.getenv('HOST', 'localhost')
 SERVER_PORT = os.getenv('PORT', 5789)
+
+DATAFRAME_OUTPUT_SAMPLE_COUNT = 10
```

Comment:
We have multiple dataframe output sample count constants. Let's put them together. https://github.com/mage-ai/mage-ai/blob/master/mage_ai/data_preparation/models/constants.py#L16-L17

Reply:
Maybe, you can update the DATAFRAME_SAMPLE_COUNT_PREVIEW to 10.